unabridged-works-volume-1/original/03-way-of-life.adoc

```diff
@@ -500,6 +502,7 @@ but lovers of the world, and of such a religion as suits the worldly spirit.
 
 ==== POSITION II
```

Comment:
Can you uncapitalize this heading, and all of the other headings like it -- in fact, just anywhere you see all-caps, go ahead and change it. They're kind of hard to notice sometimes though, i know. :)